code/modules/antagonists/changeling/powers/chameleon_skin.dm

```diff
@@ -26,4 +32,9 @@
 	if(C.dna.GetSEState(GLOB.chameleonblock))
 		C.dna.SetSEState(GLOB.chameleonblock, 0)
 		singlemutcheck(C, GLOB.chameleonblock, MUTCHK_FORCED)
+	if(C.dna.GetSEState(GLOB.noprintsblock))
+		C.dna.SetSEState(GLOB.noprintsblock, 0)
+		singlemutcheck(C, GLOB.noprintsblock, MUTCHK_FORCED)
+		if(HAS_TRAIT_FROM(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT))
+			REMOVE_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
```

Comment:
Just remove it, it shouldn't fail if it doesn't exist
```suggestion
		REMOVE_TRAIT(user, TRAIT_SILENT_FOOTSTEPS, CHANGELING_TRAIT)
```

Reply:
Change pushed.